src/ecdsa.js

```diff
@@ -243,5 +242,8 @@ module.exports = {
   deterministicGenerateK: deterministicGenerateK,
   recoverPubKey: recoverPubKey,
   sign: sign,
-  verify: verify
+  verify: verify,
+
+  // TODO: remove
+  __curve: secp256k1
```

Comment:
Necessary to avoid strange issues with `infinity` due to no de-duplication occurring by npm.